registration/src/main/java/com/kyriba/registration/api/dto/JobRequest.java

```diff
@@ -0,0 +1,23 @@
+package com.kyriba.registration.api.dto;
+
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+
+@ApiModel(description = "Request model for attendee job")
+@Data
+@NoArgsConstructor
+public class JobRequest
```

Comment:
This is not request and may be easily used in other places as Job.

Reply:
ok, renamed to JobDto